python_ta/__init__.py

```diff
@@ -314,20 +359,30 @@ def reset_linter(
     linter.load_plugin_modules(custom_checkers)
     linter.load_plugin_modules(["python_ta.transforms.setendings"])
 
+    # Logic for loading configurations:
```

Comment:
It's good to document this, but I realized you should really do it in the function docstring (since the description there is now out of date).